### PR TITLE
[FEATURE] add peak apex to output

### PR DESCRIPTION
**Pull request by hroest**
forgot to output this before

### CODE REVIEW

**Comment by hroest** [comment]
fixes #1814

File: src/topp/OpenSwathFeatureXMLToTSV.cpp [hunk 1]
```diff
@@ -279,11 +292,16 @@ void write_out_body_(std::ostream &os, Feature *feature_it, TargetedExperiment &
...
       sprintf(intensity_char, "%f", sub_it->getIntensity());
       sprintf(mz_char, "%f", sub_it->getMZ());
-      os << line << meta_values << (String)intensity_char << "\t" << Peak_Apex << "\t" << (String)sub_it->getMetaValue("native_id") << "\t" << (String)mz_char << std::endl;
+      String apex = "NA";
+      if (sub_it->metaValueExists("peak_apex_int"))
+      {
+        sprintf(intensity_apex_char, "%f", (double)sub_it->getMetaValue("peak_apex_int"));
+        apex = (String) intensity_apex_char;
+      }
+      os << line << meta_values << (String)intensity_char << "\t" << apex << "\t" << (String)sub_it->getMetaValue("native_id") << "\t" << (String)mz_char << std::endl;
```
Comment (mwalzer) [inline src/topp/OpenSwathFeatureXMLToTSV.cpp]: Just out of curiosity, in OpenSWATH, do you connect spectra via a string "native_id" or is it a numerical native id or an index to the file?

**Comment by mwalzer** [comment]
:+1: on pass

File: src/topp/OpenSwathFeatureXMLToTSV.cpp (hunk 1 above)
Comment (hroest) [inline src/topp/OpenSwathFeatureXMLToTSV.cpp]: no, "native_id" is a string that we generate in the TraML file and refers to a specific chromatogram / transition that we extract from SWATH-MS data. So it is a value that we generate ourselves and corresponds to the "id" in the TraML file.

**Comment by hroest** [comment]
lets hope it passes now ...

**Comment by hroest** [comment]
I think it passed, only mac has some weird hickup

**Pull request merged**